app/components/common/inputOne/index.tsx

```diff
@@ -0,0 +1,27 @@
+interface InputOneProps {
+  label: string;
+  placeholder: string;
+  link?: string;
+}
+
+const InputOne: React.FC<InputOneProps> = ({ label, placeholder, link }) => (
```

Comment:
Could we think of better variable name here please?
UserInput or something rather than InputOne

Reply:
okay, UserInput sounds good